packages/nextjs/app/builders/0x8Bb563DC969cC3b3e7275474c3847b44dBCB9441/page.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Image from "next/image";
+import { NextPage } from "next";
+
+const MichaelEsenwa: NextPage = () => {
```

Comment:
You might want increase the readability in light mode (dark mode is good).

<img width="713" alt="Screenshot 2024-10-31 at 11 26 11" src="https://github.com/user-attachments/assets/37993ed0-0eb7-4c0b-b853-4f7754856ba0">

Reply:
Hi @phipsae , I have made all the corrections you pointed out.